### PR TITLE
Update Gum docs

### LINKED ISSUE #605: Gum theme: Multi font title
In the screenshot for the gum theme, the first line is normal font and the second line is bolded. I've tried looking around for a title setting to accomplish this but I haven't been able to find anything. Could someone point me in the right direction?

### PR DESCRIPTION
**Pull request by rask004**
- [x] add missing variable documentation: SITESUBTITLE
- [x] add missing documentation for disqus
- [x] add standard settings honored by theme
- [x] closes #605

### CODE REVIEW

File: gum/README.md [hunk 1]
```diff
@@ -38,6 +38,27 @@ JUVIA_URL = ''
 JUVIA_ID = ''
 ```
 
+To use Discus, fill out the following values, again the url without leading `http://` and trailing `/`.
```
Comment (justinmayer) [inline gum/README.md]: Discus → Disqus

File: gum/README.md [hunk 2]
```diff
@@ -38,6 +38,27 @@ JUVIA_URL = ''
...
+DISQUS_SITENAME = ''
+```
+
+Other Features include:
+
+```
+SITESUBTITLE = 'your site subtitle'
+```
+
+The following standard pelican settings are honored:
```
Comment (justinmayer) [inline gum/README.md]: pelican → Pelican

**Comment by rask004** [comment]
Spelling fixed. Will look into rebase and squash for local workflows.
Tried to fix up the .gitmodule but not sure I've done it right.

**Comment by justinmayer** [comment]
Thanks, Roland. The .gitmodule changes are unrelated to the Gum theme documentation enhancements and thus don't belong in the same pull request. I've taken the liberty of cherry-picking your d52e6df49cf7fb9f87ec23d80440fdbcec37495e commit and adding my own text fixes via b0d1a0b1facf05805e09c7e7994ca225cd55bcea.

**Pull request closed without merging**